warehouse/templates/manage/settings.html

```diff
@@ -19,7 +19,10 @@
 
 {% block main %}
   <h2>{% trans %}Project settings{% endtrans %}</h2>
-
+  {% trans project_upload_limit=project.upload_limit | int / 1048576 , help_url=request.help_url(_anchor='file-size-limit') %}Project upload limit is: 
+  <a href="{{ help_url }}">{{ project_upload_limit }} MiB</a>{% endtrans %}<br><br>
+  {% trans project_total_limit=project.total_size_limit | int / 1073741824 , help_url=request.help_url(_anchor='project-size-limit') %}Project total size limit is: 
+  <a href="{{ help_url }}">{{ project_total_limit }} GiB</a>{% endtrans %}<br><br>
```

Comment:
You can use the [`filesizeformat`](https://jinja.palletsprojects.com/en/2.11.x/templates/#filesizeformat) filter here instead to convert these to human-readable values.

Reply:
Okay. I will use that. One more thing
Do I have to run 
$make translation 
Before committing?
It's not passing the test case.

Reply:
Yes, see https://warehouse.readthedocs.io/development/submitting-patches/#translations

Reply:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/sahil/Downloads/warehouse/warehouse/i18n/__init__.py", line 15, in <module>
    from babel.core import Locale
ModuleNotFoundError: No module named 'babel'
/.state/env/bin/isort -rc *.py warehouse/ tests/
/bin/sh: 1: /.state/env/bin/isort: not found
make: *** [Makefile:103: reformat] Error 127

I have already installed babel. I don't have any idea why this is happening.